cmd/flag.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/giantswarm/crsync/internal/env"
+	"github.com/giantswarm/microerror"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	flagDstRegistryName     = "dst.name"
```

Comment:
Can we also add `src.name` for completes?

Reply:
I didn't add it as for now the source registry is specifically quay. We're going to retag from quay to docker/acr before all the repos migrated to push directly into dockerhub